gpt_slide_explainer.py

```diff
@@ -0,0 +1,116 @@
+import sys
+from typing import Dict
+import asyncio
+import openai
+
+MAX_TOKENS = 200
+TEMPERATURE = 0.7
+
+WINDOWS_PLATFORM = 'win'
+
+
+class GPTSlideExpander:
+
+    def __init__(self):
+        GPTSlideExpander.init_api_key()
+        self.expanded_slide_explanations = {}
+        if sys.platform.startswith(WINDOWS_PLATFORM):
+            asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+
+    @staticmethod
+    def init_api_key():
+        """get the api key from the environment arguments"""
+        # openai.api_key = os.getenv("OPENAI_API_KEY")
```

Comment:
Decide what you want to do, if you read API key from file, remove the unncessery line and change the docstring

Reply:
Done